README.md

```diff
@@ -1,11 +1,26 @@
 
 <img width="800px" src="www/src/collage-top.png"/>
 
-# Chatmail servers for end-to-end encrypted instant messaging 
+# Chatmail servers for secure instant messaging 
 
-This repository helps to setup a ready-to-use chatmail server
+Chatmail servers are minimal interoperable e-mail routing machines designed for:
+
+- Convenience: Instant onboarding, with optional Google/Apple/Huawei push notifications
+
+- Privacy: Just login, no questions asked, no name, numbers or e-mail needed
+
+- Speed: End-to-End Message delivery in well under a second
+
+- Security: Strict TLS, DKIM and OpenPGP with metadata-minimization enforced. 
+
+- Relaxation: No annoying spam-checking, IP reputation or rate limits
+
+- Efficiency: messages are only stored for transit and removed automatically. 
```

Comment:
as we have a list with nice keywords, maybe make it a bit broader, and improve overall look of the README by that

```suggestion
- **Convenience:** Instant onboarding, with optional Google/Apple/Huawei push notifications

- **Privacy:** Just login, no questions asked, no name, numbers or e-mail needed

- **Speed:** End-to-End Message delivery in well under a second

- **Security:** Strict TLS, DKIM and OpenPGP with metadata-minimization enforced. 

- **Relaxation:** No annoying spam-checking, IP reputation or rate limits

- **Efficiency:** messages are only stored for transit and removed automatically. 
```